src/pymor/models/iosys.py

```diff
@@ -127,6 +134,13 @@ def __init__(self, A, B, C, D=None, E=None, sampling_time=0,
         sampling_time = float(sampling_time)
         assert sampling_time >= 0
 
+        assert presets is None or presets.keys() <= {'poles', 'c_lrcf', 'o_lrcf', 'c_dense', 'o_dense', 'hsv',
+                                                     'h2_norm', 'hinf_norm', 'l2_norm', 'linf_norm', 'fpeak'}
+        if isinstance(presets, dict) and presets:
```

Comment:
```suggestion
        if presets:
```
Checking the type does not make sense to me here. If you want, you can add an assertion above. With the current code, you could pass in a dict-like object without error, even though one of the operators is parametric.